contracts/governance/extensions/GovernorTimelockAccess.sol

```diff
@@ -92,22 +109,63 @@ abstract contract GovernorTimelockAccess is Governor {
         _baseDelay = newBaseDelay;
     }
 
+    /**
+     * @dev Check if restrictions from the associated {AccessManager} are ignored for a target function. Returns true
+     * when the target function will be invoked directly regardless of `AccessManager` settings for the function.
+     * See {setAccessManagerIgnored} and Security Considerations above.
+     */
+    function isAccessManagerIgnored(address target, bytes4 selector) public view virtual returns (bool) {
+        bool isGovernor = target == address(this);
+        return _ignoreToggle[target][selector] != isGovernor; // equivalent to: isGovernor ? !toggle : toggle
+    }
+
+    /**
+     * @dev Configure whether restrictions from the associated {AccessManager} are ignored for a target function.
+     * See Security Considerations above.
+     */
+    function setAccessManagerIgnored(
+        address target,
+        bytes4[] calldata selectors,
+        bool ignored
+    ) public virtual onlyGovernance {
+        for (uint256 i = 0; i < selectors.length; ++i) {
+            bytes4 selector = selectors[i];
+            _setAccessManagerIgnored(target, selector, ignored);
```

Comment:
If we are not reusing the selector, do we get anything from caching it?
```suggestion
            _setAccessManagerIgnored(target, selectors[i], ignored);
```

Reply:
No, this was leftover from previous code.